app/pages/system/AuditLog.tsx

```diff
@@ -22,11 +22,25 @@ import { Badge } from '~/ui/lib/Badge'
 import { Button } from '~/ui/lib/Button'
 import { PageHeader, PageTitle } from '~/ui/lib/PageHeader'
 import { Spinner } from '~/ui/lib/Spinner'
+import { classed } from '~/util/classed'
 import { toSyslogDateString, toSyslogTimeString } from '~/util/date'
 import { docLinks } from '~/util/links'
 
 export const handle = { crumb: 'Audit Log' }
 
+// todo
+// might want to still render the items in case of error
+const ErrorState = () => {
+  return <div>Error State</div>
+}
+
+// todo
```

Comment:
I think I'll do a skeleton state that is essentially a bunch of empty items to avoid too much layout shift when they load